node/src/main/scala/com/wavesplatform/state/Blockchain.scala

```diff
@@ -50,6 +51,8 @@ trait Blockchain {
   def transactionInfo(id: ByteStr): Option[(Int, Transaction)]
   def transactionHeight(id: ByteStr): Option[Int]
 
+  def portfolioNFT(address: Address, from: Option[IssuedAsset]): CloseableIterator[IssueTransaction]
```

Comment:
`nftList` seems like a better name to me: what it returns isn't actually a portfolio.